serve/graph/schema/filter/transaction_filter.graphql

```diff
@@ -65,6 +65,54 @@ input TransactionFilter {
   `success` enables you to filter between successful and unsuccessful transactions.
   """
   success: Boolean
+
+  """
+  `events` are the events that the transaction has emitted.
+  `events` filters transactions with specific events.
+  """
+  events: [TransactionEventInput!]
+}
+
+"""
+Transaction's event to filter transactions.
+"TransactionEventInput" can be configured as a filter with a transaction event's `type` and `pkg_path` and `func`, and `attrs`.
+"""
+input TransactionEventInput {
+  """
+  `type` is the type of transaction event emitted.
+  """
+  type: String
```

Comment:
Do we have specific types here? If that is the case, maybe we should change the type from String to an enumerator.

Reply:
IMHO, I don't think it's possible to specify the type of event. 
This is the type that the user inputs when using the emit function.
https://github.com/gnolang/gno/blob/master/gnovm/stdlibs/std/emit_event.go#L14-L31